test-frame-metrics-collector/MetricsCollector.md

```diff
@@ -0,0 +1,101 @@
+# MetricsCollector
```

Comment:
Why the name isn't `READMEmd` ?

Reply:
I wanted to be more specific... If that's a problem I can name it `README.md` :)

Reply:
Guess it is up to you :) I don't have any strong preference.

Reply:
Actually....when it will be named README.md it will be automatically opened when user opens the module. Maybe in that case it's worthy to change it?

Reply:
Yeah, that's worthy 100% :) Thanks